src/Servers/IIS/AspNetCoreModuleV2/CommonLib/StringHelpers.cpp

```diff
@@ -4,17 +4,6 @@
 #include "StringHelpers.h"
 #include "exceptions.h"
 
-bool ends_with(const std::wstring &source, const std::wstring &suffix, bool ignoreCase)
```

Comment:
we now get an ends_with pulled in from libnethost

Reply:
Or namespace ours

Reply:
Just curious: What goes South if you remove ours and use the libnethost one❔